README.md

```diff
@@ -689,6 +689,7 @@ Clients for commercial social platforms that had their API access cut off in a w
 
 #### Simple Editors and Light IDEs
 
+- [Bluefish](https://bluefish.openoffice.nl) - Editor targeted towards programmers and web developers (C, JavaScript, Java, PHP, Python, markup languages HTML, YAML and XML) `#c` `#gtk3`.
```

Comment:
```suggestion
- [Bluefish](https://bluefish.openoffice.nl) - Editor targeted towards programmers and web developers (C, JavaScript, Java, PHP, Python, markup languages, HTML, YAML and XML) `#c` `#gtk3`.
```

Reply:
I shortened the text by Bluefish which was: ... as well as markup languages such as HTML, YAML and XML
Therefore, I suggest to use a colon ":" instead of a comma before HTML.